### DISCUSSION TITLE
Any way to familiarize myself with the Datalog language?

### DISCUSSION THREAD

**Discussion by heedongcho-xyz** [cozodb/cozo#130] [General]
It's really my first time being exposed to a logic-based query language so I'm wondering if there'd be any good resources I could learn from.

**Comment by cwarden** [comment]
There are a number of university lecture presentation slides online if you search for "datalog slides".  I learned enough using those and the [souffle documentation](https://souffle-lang.github.io/) to be able to solve problems using datalog.

**Comment by davidhmays** [comment]
These are for the Datomic and XTDB dialects of Datalog but could be helpful:

https://www.learndatalogtoday.org/
https://nextjournal.com/try/learn-xtdb-datalog-today/learn-xtdb-datalog-today
